optuna/samplers/_cmaes.py

```diff
@@ -373,6 +400,17 @@ def _init_optimizer(
         sigma0 = max(sigma0, _EPS)
         bounds = _get_search_space_bound(ordered_keys, search_space)
         n_dimension = len(ordered_keys)
+
+        if self._use_separable_cma:
+            return SepCMA(
+                mean=mean,
+                sigma=sigma0,
+                bounds=bounds,
+                seed=self._cma_rng.randint(1, 2 ** 31 - 2),
+                n_max_resampling=10 * n_dimension,
+                population_size=population_size,
+            )
```

Comment:
I don't think the current test cases cover this logic. How about updating the `test_init_cma_opts`, for example, as follows?

```python
@pytest.mark.parametrize("use_separable_cma, cma_class_str", 
    [(False, "optuna.samplers._cmaes.CMA"), (True, "optuna.samplers._cmaes.SepCMA")])
def test_init_cmaes_opts(use_separable_cma: bool, cma_class_str: str) -> None:
    sampler = optuna.samplers.CmaEsSampler(
        x0={"x": 0, "y": 0},
        sigma0=0.1,
        seed=1,
        n_startup_trials=1,
        independent_sampler=DeterministicRelativeSampler({}, {}),
        use_separable_cma=use_separable_cma
    )
    study = optuna.create_study(sampler=sampler)

    with patch(cma_class_str) as cma_class:
        cma_obj = MagicMock()
        cma_obj.ask.return_value = np.array((-1, -1))
        cma_obj.generation = 0
        cma_obj.population_size = 5
        cma_class.return_value = cma_obj
        study.optimize(
            lambda t: t.suggest_uniform("x", -1, 1) + t.suggest_uniform("y", -1, 1), n_trials=2
        )

        assert cma_class.call_count == 1

        _, actual_kwargs = cma_class.call_args
        assert np.array_equal(actual_kwargs["mean"], np.array([0, 0]))
        assert actual_kwargs["sigma"] == 0.1
        assert np.allclose(actual_kwargs["bounds"], np.array([(-1, 1), (-1, 1)]))
        assert actual_kwargs["seed"] == np.random.RandomState(1).randint(1, 2 ** 32)
        assert actual_kwargs["n_max_resampling"] == 10 * 2
        assert actual_kwargs["population_size"] is None
```